src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -283,14 +283,14 @@ public void WriteTo(TextWriter writer, HtmlEncoder encoder)
 
             if (!isTagNameNullOrWhitespace)
             {
-                writer.Write('<');
+                writer.Write("<");
```

Comment:
Changing these to string to avoid boxing in the common case (`HtmlTextWriter` that stores string or `IHtmlContent`).